src/imitation/bc.py

```diff
@@ -36,7 +36,7 @@ def __init__(self,
     self.env = env
     self.policy_class = policy_class
     self.batch_size = batch_size
-    expert_obs, expert_acts, _ = expert_rollouts
+    expert_obs, expert_acts, _, _ = expert_rollouts
```

Comment:
If `Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]` is too long for your taste, could use `Tuple[np.ndarray, ...]` to fix pytype error